### PR TITLE
chore: Reorganize graphql queries types in BFF

### LINKED ISSUE #1255: Chore: Reorganize queries and mutations types in BFF
The way the queries and mutations in `./graphql/types/*` are organized is a  quite confusing in terms order and hierarchy. 
Suggestion to new way of organizing the types:

- Echo `queries.ts` by having a `mutations.ts` with all definition with `type: 'Mutation'` .
- Move `queries.ts` and `mutations.ts` one level up, i.e. to root of `graphql`.
- Ensure all types are being used (delete if unnecessary)
- Ensure all mutations / queries are being used by client (delete if unecessary)

### PR DESCRIPTION
**Pull request by mbacherycz**
Reorganize graphql queries types in bff and move mutations types generators to its own file 

Closes #1255

### Dokumentasjon / Storybook / testdekning

- [ ] Dokumentasjon er oppdatert eller ikke relevant / nødvendig.
- [ ] Ny komponent har en eller flere `stories` i `Storybook`, eller så er ikke dette relevant.
- [ ] Det er blitt lagt til nye tester / eksiterende tester er blitt utvidet, eller tester er ikke relevant.

## Summary by CodeRabbit

- **New Features**
	- Added a new `Response` object type to provide feedback on mutation success.

- **Improvements**
	- Simplified resolver function signatures across various object types for better clarity and maintainability.
	- Enhanced module interface by consolidating exports from the mutation module.

- **Bug Fixes**
	- Removed unused parameters from resolver functions to streamline operations.
  
- **Chores**
	- Removed mutation fields from the `Mutation` type to reduce complexity in operations.

### CODE REVIEW

**Comment by seanes** [comment]
Conflict with pnpm-lock, but I don't understand why there should be any changes to the lock file. Rebase against latest on main and run `pnpm i` again?

**Comment by seanes** [comment]
Other than that, it looks good to me :)

File: packages/bff/src/graphql/types/savedSearches.ts [hunk 1]
```diff
@@ -108,32 +35,32 @@ export const SavedSearches = objectType({
   definition(t) {
     t.int('id', {
       description: 'id of savedSearch',
-      resolve: (source, args, ctx, info) => {
+      resolve: (source) => {
         return source.id;
       },
```
[the remark this answers is not in the document]
Comment (seanes) [reply packages/bff/src/graphql/types/savedSearches.ts:40]: @mbacherycz  ☝️  This might be true, yes, haven't verified if the rabbit is guessing or not :)

**Comment by seanes** [comment]
Rename `query.ts´ to `queries.ts`, perhaps? A bit inconsistent now.
Also, maybe move `queries.ts` and `mutations.ts` up a level?

**Comment by mbacherycz** [comment]
> Rename `query.ts´ to `queries.ts`, perhaps? A bit inconsistent now. Also, maybe move `queries.ts`and`mutations.ts` up a level?

Query is an objectType, it returns a type of name Query so I think it makes sense to leave this name as it matches the definition. 
Not sure what you mean by moving these two files up 🤔 They are the same lvl as index where they're being imported.

**Review by seanes — APPROVED** [review]

**Pull request merged**